src/Sylius/Bundle/ShippingBundle/spec/Form/Type/RuleChoiceTypeSpec.php

```diff
@@ -22,8 +22,8 @@
 class RuleChoiceTypeSpec extends ObjectBehavior
 {
     private $choices = [
-        RuleInterface::TYPE_ITEM_TOTAL => 'Order total',
-        RuleInterface::TYPE_ITEM_COUNT => 'Order items count',
+        RuleInterface::TYPE_UNIT_TOTAL => 'Order total',
```

Comment:
I am not sure about it. This should be probably renamed to ITEMS_TOTAL, no? Units in that context can be confusing?